docs/cep/query-guide/query.md

```diff
@@ -302,44 +302,44 @@ Windows help to retain events based on a criterion, such that the values of thos
 
 **Syntax**
```

Comment:
Avoid using bold text in place of headings. In all cases, change the bold text pretending to be headings to actual headings.